pkg/agent/openflow/pipeline.go

```diff
@@ -594,17 +594,16 @@ func (f *featurePodConnectivity) gatewayClassifierFlow() binding.Flow {
 
 // podClassifierFlow generates the flow to mark the packets from a local Pod port.
 func (f *featurePodConnectivity) podClassifierFlow(podOFPort uint32, isAntreaFlexibleIPAM bool) binding.Flow {
-	flowBuilder := ClassifierTable.ofTable.BuildFlow(priorityLow).
-		Cookie(f.cookieAllocator.Request(f.category).Raw()).
-		MatchInPort(podOFPort).
-		Action().LoadRegMark(FromLocalRegMark).
-		Action().GotoStage(stageValidation)
+	regMarksToLoad := []*binding.RegMark{FromLocalRegMark}
 	if isAntreaFlexibleIPAM {
-		// This is used to mark the packets from a local Antrea IPAM Pod port.
-		flowBuilder = flowBuilder.Action().LoadRegMark(AntreaFlexibleIPAMRegMark).
-			Action().LoadRegMark(RewriteMACRegMark)
+		regMarksToLoad = append(regMarksToLoad, AntreaFlexibleIPAMRegMark, RewriteMACRegMark)
```

Comment:
Is this required? I think ofnet already implemented this: https://github.com/antrea-io/ofnet/pull/20